### PR TITLE
Add `Commented` button in the main issues search tab

### LINKED ISSUE #1102: Add 'Commented' button in the main issues search tab
Similar to [this](https://user-images.githubusercontent.com/170270/27501170-f394a304-586b-11e7-92d8-d92d6922356b.png), but for the [github.com/issues tab](https://github.com/issues). So that we would have Created/Assigned/Mentioned/Yours/Commented, or other useful statuses.

### PR DESCRIPTION
**Pull request by tanmayrajani**
Fixes #1102

### CODE REVIEW

**Comment by bfred-it** [comment]
Thsi works, however I'd like to see it on the PRs too. Can you try to fit them on one line with some CSS (`display:flex`) like this?

one line

Alternative solutions welcome

File: source/content.js [hunk 1]
```diff
@@ -171,6 +170,8 @@ function ajaxedPagesHandler() {
 		enableFeature(addCommentedMenuItem);
 	}
 
+	enableFeature(sortIssuesByUpdateTime); // Must be before addYoursMenuItem + addCommentedMenuItem
+
```
Comment (tanmayrajani) [inline source/content.js]: @bfred-it you sure the comment is correct here? I see you moved `sortIssuesByUpdateTime` **after** the `addYoursMenuItem` + `addCommentedMenuItem` but the comment says _**before**_ :smile: 

Or I might have misunderstood..

Comment (bfred-it) [reply source/content.js]: Oops.

**Comment by tanmayrajani** [comment]
- We can decrease the size of the search bar further (I see we have done that already 
for `Yours` filter)
image
- Decrease the padding of all filter buttons to accommodate them in one line
image

- Or a bit of both?

**Comment by bfred-it** [comment]
If you can do just the first one I think that's ok

**Comment by tanmayrajani** [comment]
@bfred-it please verify when you get chance. thanks.

**Comment by bfred-it** [comment]
Let's see if anyone else has anything to say about the style

**Comment by busches** [comment]
The screenshots look identical to me, so I'm ok with either. 👍

**Comment by bfred-it** [comment]
Can you mention it in the readme? It can be mentioned on the same line for `Yours`

**Comment by tanmayrajani** [comment]
Here's the final screenshot. Adding it to readme.
image

File: source/content.css [hunk 2]
```diff
@@ -577,6 +577,10 @@ div.inline-comment-form .form-actions,
 	width: 420px;
 }
 
+.subnav-links .subnav-item {
```
Comment (sindresorhus) [inline source/content.css]: This needs a comment on why it's there and what it does.

Comment (sindresorhus) [reply source/content.css]: I think it should also be moved to a more relevant place in the file: https://github.com/tanmayrajani/refined-github/blob/27fc4a338a8e27f7e84cbd2de1b7cb4499569755/source/content.css#L575

File: source/features/add-commented-menu-item.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import select from 'select-dom';
+import * as pageDetect from '../libs/page-detect';
+import {getUsername} from '../libs/utils';
+
+export default function () {
+	const pageName = pageDetect.isIssueSearch() ? 'issues' : 'pulls';
+	const type = pageDetect.isIssueSearch() ? 'issue' : 'pr';
+	const username = getUsername();
+
+	if (select.exists('.refined-github-commented')) {
```
Comment (sindresorhus) [inline source/features/add-commented-menu-item.js]: This should be at the top of the function, so we do the least amount of work if it already exists.

**Pull request merged**